apps/extension/src/App/Accounts/AddAccount.tsx

```diff
@@ -139,21 +139,24 @@ const AddAccount: React.FC<Props> = ({ accounts, requester, setAccounts }) => {
                 {[bip44Prefix, coinType].join("/")}/
               </Bip44PathDelimiter>
               <Bip44Input
-                type="text"
+                type="number"
```

Comment:
Note that this form has changed a quite a bit in #97, to match closely match the form on polkadot-js extensino: https://github.com/anoma/namada-interface/blob/1bcb3230792fb0d72f27e962f462b6272668783a/apps/extension/src/App/Accounts/AddAccount.tsx#L142-L167 Until we have a more specific requirement, it makes sense to do it this way as it enforces _valid_ derivation paths under the mnemonic seed.

Reply:
Specifically, when deriving an address from an account on polkadot, the user can specify `change` and `index`. `change` can only be a 0 (external address) or 1 (internal), and `account` can be any unused value (though it's preferable to be sequential) See https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#Change